openpype/hosts/traypublisher/plugins/publish/collect_review_frames.py

```diff
@@ -16,7 +16,7 @@ class CollectReviewInfo(pyblish.api.InstancePlugin):
 
     label = "Collect Review Info"
     order = pyblish.api.CollectorOrder + 0.491
-    families = ["review"]
+    families = ["review", "render"]
```

Comment:
You should add "review" family to instance if you want to trigger this plugin.
```suggestion
    families = ["review"]
```

Reply:
One could argue that `plugin requires frame start/end, fps on instance data which are missing on instances from TrayPublishes.` isn't just for review purposes, what if I don't want a review?

Reply:
I believe this issue is actually similar to another PR currently open. Does https://github.com/ynput/OpenPype/pull/5152 solve that too @fabiaserra ?

_Sorry if I'm incorrect - a bit in a hurry myself_

Reply:
yeah I believe so too, that's why I commented there this https://github.com/ynput/OpenPype/pull/5152#discussion_r1240099456 a few days ago as I had just encountered I needed this plugin, I think they should both be aligned for the same purpose

Reply:
Ok then let's handle that in that PR and keep this PR without that?

Reply:
Sure